data/io.github.fsobolev.Cavalier.gschema.xml

```diff
@@ -49,6 +50,12 @@
 			<range min="0" max="50"/>
 			<default>10</default>
 		</key>
+	  	<key name="line-thickness" type="i">
+			<summary>Thickness of lines</summary>
+			<description>Thickness of lines in "line" mode (in pixels).</description>
+			<range min="5" max="30"/>
```

Comment:
I think that minimal value should be 1, to get a very thin line

![](https://i.imgur.com/GbCBN7l.png)